content/participate/code.adoc

```diff
@@ -1,27 +1,100 @@
 ---
-layout: simplepage
+layout: project
 title: Contribute code
 section: participate
+tags:
+  - outreach-programs
+  - developer
 ---
 
-[NOTE]
-====
-This page is still a work in progress.
-====
+Jenkins project includes a lot of code, and we invite everyone to contribute to the project.
+There is a diverse set of programming languages used in Jenkins,
+including but not limited to: Java, JavaScript, Groovy, Golang, Ruby, Shell scripts.
+And, since Jenkins is an automation server with hundreds of plugins, there is a huge number of technologies involved.
+If you are an expert or just want to study something new while contributing,
+you may find interesting opportunities in our project.
 
-Check out these existing resources for contributors:
+== Where to contribute?
 
-* link:https://wiki.jenkins.io/display/JENKINS/Beginners+Guide+to+Contributing#BeginnersGuidetoContributing-Areyouinterestedinwritingcode%3F[Beginners Guide to Contributing].
-* https://wiki.jenkins.io/display/JENKINS/Hosting+Plugins[The complete guide to hosting and publishing plugins]
+The Jenkins project is spread across multiple organizations on GitHub.
+You are welcome to contribute to **any** repository in **any** of those organizations, or to any other Jenkins-related repository on GitHub.
+
+* https://github.com/jenkinsci[jenkinsci] - Main organization. Jenkins core, plugins and libraries reside there
+* https://github.com/jenkins-infra[jenkins-infra] - Jenkins infrastructure, including the website and other services
+* https://github.com/stapler/[stapler] - Stapler Web Framework which is currently maintained by the Jenkins community
+* https://github.com/jenkins-zh[jenkins-zh] - organization of the link:/sigs/chinese-localization/[Chinese Localization SIG]
```

Comment:
Shouldn't we also add https://github.com/jenkins-docs

Reply:
Did not know it exists :D 
CC @MarkEWaite what is the plan about it? Should we get it listed?

Reply:
The repositories in the jenkins-docs organization are used for demonstrations and tutorials in the Jenkins documentation.  The Blue Ocean tutorial uses the https://github.com/jenkins-docs/simple-java-maven-app repository./